Cargo.toml

```diff
@@ -6,7 +6,6 @@ members = ["examples/with_winit", "examples/with_bevy", "examples/run_wasm"]
 [workspace.package]
 edition = "2021"
 version = "0.1.0"
-authors = ["piet-gpu developers"]
```

Comment:
There are authors also set elsewhere, which should probably also be removed

That is at least further down this file.

Reply:
Ok, I removed the authors key in the [package] section, which is I think what you mean.